### ISSUE TITLE
Unique Last Line not Considered on files with CRLF line endings

### ISSUE THREAD

**Issue by kylezimmerman** [atom/sort-lines#51]
When using the *Lines > Unique* command the last line appears to not be checked for uniqueness.

For example if I have a file:
```
A
B
B
A
```
I would expect to get:
```
A
B
```

However, I get:
```
A
B
A
```

The [spec](https://github.com/atom/sort-lines/blob/master/spec/sort-lines-spec.coffee) looks like it fails as well.

```coffee
describe "uniqueing", ->
    it "uniques all lines but does not change order", ->
      editor.setText """
        Hydrogen
        Hydrogen
        Helium
        Lithium
        Hydrogen
        Hydrogen
        Helium
        Lithium
        Hydrogen
        Hydrogen
        Helium
        Lithium
        Hydrogen
        Hydrogen
        Helium
        Lithium
      """

      editor.setCursorBufferPosition([0, 0])

      uniqueLines ->
        expect(editor.getText()).toBe """
          Hydrogen
          Helium
          Lithium
        """
```
But it results in:
```
Hydrogen
Helium
Lithium
Lithium
```

**Comment by lee-dohm** [comment]
Running the specs using Atom v1.4.1 on Mac OS X 10.11.3 shows no failures for me. Could you give the following information?

* Which version of Atom are you using?
* What OS and version are you running?
* Which version of sort-lines do you have installed?

**Comment by lee-dohm** [comment]
I also don't get the behavior you specify.

**Comment by kylezimmerman** [comment]
Sure thing
- **Atom Version:** 1.4.1
- **OS:** Windows 10 x64 (Version 1511)
- **sort-lines Version:** 0.13.0

**Comment by lee-dohm** [comment]
I assume you're using CRLF terminated lines?

**Comment by lee-dohm** [comment]
Yep ... that's where the bug reproduces.

**Comment by kylezimmerman** [comment]
This should be pretty simple to solve, I'll submit a PR.

**Comment by lee-dohm** [comment]
Awesomesauce! Thanks :grinning:

**Comment by lee-dohm** [comment]
Closed in #52
